db/crud.go

```diff
@@ -1106,13 +1106,25 @@ func (db *DatabaseCollectionWithUser) PutExistingCurrentVersion(ctx context.Cont
 			if addNewerVersionsErr != nil {
 				return nil, nil, false, nil, addNewerVersionsErr
 			}
+			if docHLV.MergeVersions != nil {
+				if doc.HLV.MergeVersions == nil {
+					doc.HLV.MergeVersions = make(map[string]uint64)
+				}
```

Comment:
I don't think this is nessesary if we're always overwriting the map in its entirety with `docHLV.MergeVersions` ?

```suggestion
```